src/brain/issueLabelHandler/route.ts

```diff
@@ -167,7 +173,6 @@ export async function markRouted({
 
   const reasonsToSkip = [
     isNotInARepoWeCareAboutForRouting,
-    isFromABot,
```

Comment:
this is being removed since we need our bot to be able to route issues if a product area field changes

Reply:
So in other words when a product area project field changes, @getsantry will apply the corresponding product area label (via the new `syncLabelsWithProjectField` handler), and applying the product area label lands us in here, and we want to apply `Status: Untriaged` and `Waiting for: Product Owner`?

Reply:
Yes, that's correct

Reply:
I feel like that's going to get annoying, it's already annoying when someone changes the product area and does not intend to re-enter the triage queue. I kinda think we should totally decouple the two, `Product Area: Foo` and `Waiting for: Bar` should be always be two operations. Wdyt? And what should we do here?

Reply:
I'm assuming that the unlabeling/labeling and comment from the bot is what's annoying, and you have a point here. I do want to completely remove the comments from the getsantry bot about when the issue is due by and instead use a field on the project for that timestamp to avoid this kind of noise in the future. Alongside that, when the `Status: Untriaged` labels go away, that will further reduce that noise. 

Do you think it needs to be addressed now? Are you saying that the bot shouldn't care about rerouting if the product area project field changes?

Reply:
I don't mind the comment, I think it's good communication with the user. The annoying part is when setting a product area automatically retriggers triage, that's not always intended.

Reply:
Yeah, this is a totally fair point given we've moved away from teams to use product areas. This might be a larger code refactor, do you think this can this be revisited after this PR is in?

Reply:
Yes, agreed, let's ship as-is and revisit down the line.